tests/pytests/unit/modules/test_win_useradd.py

```diff
@@ -0,0 +1,344 @@
+    groups = ["Backup Operators"]
+    ret = user.chgroups(account_int.username, groups=groups)
+    assert ret is True
+    ret = user.info(account_int.username)
+    groups.append("Users")
+    assert salt.utils.data.compare_lists(ret["groups"], groups) == {}
+
+
+def test_chgroups_list_str(account_str):
+    groups = ["Backup Operators", "Guests"]
+    ret = user.chgroups(account_str.username, groups=groups)
+    assert ret is True
+    ret = user.info(account_str.username)
+    groups.append("Users")
+    assert salt.utils.data.compare_lists(ret["groups"], groups) == {}
+
+
+def test_chgroups_list_int(account_int):
+    groups = ["Backup Operators", "Guests"]
+    ret = user.chgroups(account_int.username, groups=groups)
+    assert ret is True
+    ret = user.info(account_int.username)
+    groups.append("Users")
+    assert salt.utils.data.compare_lists(ret["groups"], groups) == {}
+
+
+def test_chgroups_list_append_false_str(account_str):
+    groups = ["Backup Operators", "Guests"]
+    ret = user.chgroups(account_str.username, groups=groups, append=False)
+    assert ret is True
+    ret = user.info(account_str.username)
+    assert salt.utils.data.compare_lists(ret["groups"], groups) == {}
+
+
+def test_chgroups_list_append_false_int(account_int):
+    groups = ["Backup Operators", "Guests"]
+    ret = user.chgroups(account_int.username, groups=groups, append=False)
+    assert ret is True
+    ret = user.info(account_int.username)
+    assert salt.utils.data.compare_lists(ret["groups"], groups) == {}
+
+
+def test_chhome_str(account_str):
+    home = r"C:\spongebob\squarepants"
+    ret = user.chhome(name=account_str.username, home=home)
+    assert ret is True
+    ret = user.info(name=account_str.username)
+    assert ret["home"] == home
+
+
+def test_chhome_int(account_int):
+    home = r"C:\spongebob\squarepants"
+    ret = user.chhome(name=account_int.username, home=home)
+    assert ret is True
+    ret = user.info(name=account_int.username)
+    assert ret["home"] == home
+
+
+def test_chprofile_str(account_str):
+    profile = r"C:\spongebob\squarepants"
+    ret = user.chprofile(name=account_str.username, profile=profile)
+    assert ret is True
+    ret = user.info(name=account_str.username)
+    assert ret["profile"] == profile
+
+
+def test_chprofile_int(account_int):
+    profile = r"C:\spongebob\squarepants"
+    ret = user.chprofile(name=account_int.username, profile=profile)
+    assert ret is True
+    ret = user.info(name=account_int.username)
+    assert ret["profile"] == profile
+
+
+def test_delete_str(account_str):
+    ret = user.delete(name=account_str.username)
+    assert ret is True
+    assert user.info(name=account_str.username) == {}
+
+
+def test_delete_int(account_int):
+    ret = user.delete(name=account_int.username)
+    assert ret is True
+    assert user.info(name=account_int.username) == {}
+
+
+def test_getUserSig_str(account_str):
+    ret = user.getUserSid(account_str.username)
```

Comment:
Why did we add a camel cased function name to a salt module I  the first place?!?!